logging/src/commonMain/kotlin/io/spine/logging/Logger.kt

```diff
@@ -29,44 +29,80 @@ package io.spine.logging
 import io.spine.logging.LoggingFactory.loggingDomainOf
 import kotlin.reflect.KClass
 
+/**
+ * Base class for fluent API loggers. A logger is a factory of fluent logging [API] instances,
```

Comment:
Let's move the portion starting with "A logger..." to a new paragraph.

Just to keep to our convention, by which the first paragraph should have just a single sentence.